integrationtest/package.json

```diff
@@ -1,11 +1,12 @@
 {
   "dependencies": {
     "@hyperledger/fabric-chaincode-integration": "^0.8.4",
-    "@hyperledger/fabric-gateway": "^1.5.0"
+    "@hyperledger/fabric-gateway": "^1.5.0",
+    "form-data": "^4.0.4"
```

Comment:
The integrationtest package does not have a direct dependency on **form-data**, and does not need one. We should not add an unnecessary direct dependency. Requirements on transitive dependency resolution should be added to the `overrides` section instead.